website/docs/plugins/ax_sweeper.md

```diff
@@ -28,36 +28,36 @@ defaults:
 ```
 
 We include an example of how to use this plugin. The file <GithubLink to="plugins/hydra_ax_sweeper/example/banana.py">example/banana.py</GithubLink>
-implements the [Rosenbrock function (aka Banana function)](https://en.wikipedia.org/wiki/Rosenbrock_function). 
+implements the [Rosenbrock function (aka Banana function)](https://en.wikipedia.org/wiki/Rosenbrock_function).
 The return value of the function should be the value that we want to optimize.
 
 
 To compute the best parameters for the Banana function, clone the code and run the following command in the `plugins/hydra_ax_sweeper` directory:
 
 ```
-python example/banana.py -m 'banana.x=int(interval(-5, 5))' 'banana.y=interval(-5, 10.1)'
+python example/banana.py -m 'banana.x=int(interval(-5, 5))' 'banana.y=interval(-5, 10.1)' hydra.sweeper.ax_config.is_noisy=false
```

Comment:
The default for this example is is_noisy=false, no need to override it here.

Reply:
replicated here for clarity per [this comment](https://github.com/facebookresearch/hydra/pull/1395#discussion_r575588955) from @shagunsodhani

Reply:
Either way works. I just think it's not adding clarity to the example.
@shagunsodhani, what do you think?

Reply:
Ahh I suggested it earlier because we were/are setting the default of `is_noisy` to `True`. Given that we are overriding it in the config, I think we can remove it.

Reply:
But that was always the case. @shagunsodhani your comment:

> probably better to override and show in the docs

suggests that it's better that the user sees this option explicitly used in the docs.

Reply:
I think we should just ship this.